sycl/include/CL/sycl/detail/pi.h

```diff
@@ -278,6 +279,8 @@ typedef enum {
       CL_DEVICE_CROSS_DEVICE_SHARED_MEM_CAPABILITIES_INTEL,
   PI_DEVICE_INFO_USM_SYSTEM_SHARED_SUPPORT =
       CL_DEVICE_SHARED_SYSTEM_MEM_CAPABILITIES_INTEL,
+  // Intel UUID extension.
+  PI_DEVICE_INFO_UUID = CL_DEVICE_UUID_KHR,
```

Comment:
In addition to this the value should be described for piDeviceGetInfo (and what it returns).
And also, please make sure that OpenCL/CUDA plugins would do smth meaningful for it.